.cirrus.yml

```diff
@@ -131,3 +123,53 @@ windows_task:
     %container_cmd% -l -c "cd $(echo '%cd%') && make V=1 install && PATH=/usr/bin:/bin find . -type f -path '*/.libs/*.exe' -print -execdir mv -t .. {} \;"
   test_script: |
     %container_cmd% -l -c "cd $(echo '%cd%') && make V=1 TFLAGS='!IDN !SCP ~612 ~1056 %tests%' test-ci"
+    
+
+task:
+  name: macOS normal
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-xcode:latest
+
+  env:
+    CIRRUS_CLONE_DEPTH: 10
+    MAKE_FLAGS: -j 4
+    CFLAGS: "-Wno-vla -mmacosx-version-min=10.9"
+
+  pkginstall_script:
+    - brew update
+    - echo libtool autoconf automake pkg-config | xargs -Ix -n1 echo brew '"x"' > /tmp/Brewfile
+    - brew update && for i in 1 2 3; do brew bundle install --no-lock --file /tmp/Brewfile && break || sleep 1; done
+  configure_script:
+    - autoreconf -fi
+    - ./configure --enable-warnings --enable-werror --without-ssl --enable-websockets
+  compile_script:
+    - make V=1 && make V=1 examples && cd tests && make V=1
+  test_script:
+    - sudo make V=1 test-ci
+  install_script:
+    - sudo make V=1 install
+
+
+task:
```

Comment:
Since this task has quite some duplication with the previous one, maybe use a matrix like the other tasks?